openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -248,9 +246,6 @@ def _get_user_info_cookie_data(request, user):
 
 def _create_and_set_jwt_cookies(response, request, cookie_settings, user=None, refresh_token=None):
     """ Sets a cookie containing a JWT on the response. """
-    if not JWT_COOKIES_FLAG.is_enabled():
-        return
-
```

Comment:
@nasthagiri @robrap I'm wondering if we should make the code below more resilient to the absence of the login `Application` model. If it doesn't exist, let's just not set the cookies instead of blowing up.

Reply:
Or to add to that thought: allow the `Application` model to not exist when running unit tests - but require it for bokchoy and running in devstack/production/etc.